### PR TITLE
Add verify-crd-upgrade script

### PR DESCRIPTION
**Pull request by qiujian16**

### CODE REVIEW

File: hack/verify-crds-version-upgrade.sh [hunk 1]
```diff
@@ -0,0 +1,93 @@
+#!/bin/bash
+
```
Comment (sttts) [inline hack/verify-crds-version-upgrade.sh:2]: write highlevel what this does

**Comment by deads2k** [comment]
The script looks ok-ish to me.  Any comments from @sttts and @damemi should be fixed as followups.  This is catching problems in already merged PRs and in PRs currently open trying to make this switch.  I want to stop teh bleeding.

/lgtm
/approve

**Comment by soltysh** [comment]
I'm manually adding valid-label, since this is not changing code, but rather adds additional layer of checks which ensures the completeness and correctness of the manifests.

**Pull request merged**